templates/docs/patterns/quote-wrapper/index.md

```diff
@@ -0,0 +1,68 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Quote Wrapper | Patterns
+---
+
+{% from "docs/macros/patterns/wip-notice.jinja" import pattern_wip_notice %}
+
+{{- pattern_wip_notice() }}
+
+A quote wrapper is a pattern used to prominently display a quotation alongside an optional citation, logo, and image.
+
+The quote wrapper pattern is composed of the following elements:
+
+| Element              | Description                                                                                 |
+| -------------------- | ------------------------------------------------------------------------------------------- |
+| Quote (**required**) | Quote text                                                                                  |
+| Header               | Header text & CTA. Useful for adding a title/link that describes all of a series of quotes. |
+| Signpost image       | Small image to display at the left/top of the quote, such as a logo or a portrait.          |
+| Call to action       | A [CTA block](/docs/patterns/cta-block)                                                     |
+| Image                | Image to associate with the quote                                                           |
+| Source               | Name, title, and organisation of the quote's source                                         |
+
+All examples shown here use `<h2>`-sized quote text. However, this should be changed according to text length.
+For short quotes, use `.p-heading--2`.
```

Comment:
Should we document heading sizes here, or macro size values?

Reply:
In the [PR that shows macro calls in code snippets](https://github.com/canonical/vanilla-framework/pull/5321), we show HTML by default still. So, I figure the best way to do this is to document the HTML classes in the MD sections, and document the Jinja choices using table of params/slots as with #5325

Reply:
Updated to use macro param/slot documentation per established standards from hero and tiered list